plugins/search-react/src/components/SearchBar/SearchBar.test.tsx

```diff
@@ -288,76 +285,52 @@ describe('SearchBar', () => {
         ]}
       >
         <SearchContextProvider>
-          <SearchBar />
+          <SearchBar debounceTime={0} />
         </SearchContextProvider>
       </TestApiProvider>,
     );
 
-    const textbox = await screen.findByRole('textbox', { name: 'Search' });
+    const textbox = screen.getByLabelText('Search');
```

Comment:
After fixing the other condition, does avoiding findByRole still make a measurable change in runtime? This one was a surprise for me for sure!

Reply:
I've ran the tests again with both options but didn't find a clear and relevant difference between both. I'm using `yarn test plugins/search-react/src/components/SearchBar/SearchBar` and changing between both implementations.
But I see no reason to change it back. Do you have any worries keeping this line?

Reply:
No, no worries. Just use what you feel is the clearest